web/viewer.js

```diff
@@ -1885,6 +1984,23 @@ window.addEventListener('keydown', function keydown(evt) {
           handled = false;
         }
         break;
+      case 80: // p
+//#if FIREFOX || MOZCENTRAL
+//      // Ctrl + Shift + P = new private window.
+//      var shouldHandlePKey = !evt.shiftKey;
+//#endif
+//#if CHROME
+//      var shouldHandlePKey = true;
+//#endif
+//#if !(FIREFOX || MOZCENTRAL || CHROME)
+        var shouldHandlePKey = !evt.shiftKey || window.chrome || window.opera;
+//#endif
+        if (shouldHandlePKey) {
+          PDFViewerApplication.requestPrint();
+          evt.stopImmediatePropagation();
```

Comment:
need `evt.preventDefault();` to stop triggering bowser's default printing behaviour